src/tally.F90

```diff
@@ -691,12 +691,134 @@ subroutine score_general_ce(p, t, start_index, filter_index, i_nuclide, &
         ! Set the delayedgroup filter index
         dg_filter = t % find_filter(FILTER_DELAYEDGROUP)
 
-        if (survival_biasing) then
-          ! No fission events occur if survival biasing is on -- need to
-          ! calculate fraction of absorptions that would have resulted in
-          ! delayed-nu-fission
-          if (micro_xs(p % event_nuclide) % absorption > ZERO .and. &
-               nuclides(p % event_nuclide) % fissionable) then
+        if (t % estimator == ESTIMATOR_ANALOG) then
+          if (survival_biasing) then
+            ! No fission events occur if survival biasing is on -- need to
+            ! calculate fraction of absorptions that would have resulted in
+            ! delayed-nu-fission
+            if (micro_xs(p % event_nuclide) % absorption > ZERO .and. &
+                 nuclides(p % event_nuclide) % fissionable) then
+
+              ! Check if the delayed group filter is present
+              if (dg_filter > 0) then
+                select type(filt => t % filters(dg_filter) % obj)
+                type is (DelayedGroupFilter)
+
+                  ! Loop over all delayed group bins and tally to them
+                  ! individually
+                  do d_bin = 1, filt % n_bins
+
+                    ! Get the delayed group for this bin
+                    d = filt % groups(d_bin)
+
+                    ! Compute the yield for this delayed group
+                    yield = nuclides(p % event_nuclide) &
+                         % nu(E, EMISSION_DELAYED, d)
+
+                    associate (rxn => nuclides(p % event_nuclide) % &
+                         reactions(nuclides(p % event_nuclide) % index_fission(1)))
```

Comment:
Why is the argument to index_fission always 1? That mens you are using MT=18 data right? I forget, do we not put the yields on the partial fissions?